perf/lmbench.py

```diff
@@ -95,12 +96,11 @@ def setUp(self):
 
         # patch the resulted config to use the proper temporary directory and
         # file locations
-        tmp_config_file = config_file + '.tmp'
-        process.system('touch ' + tmp_config_file)
-        process.system("sed 's!^FSDIR=.*$!FSDIR=%s!' '%s'  '%s' " %
-                       (fsdir, config_file, tmp_config_file))
-        process.system("sed 's!^FILE=.*$!FILE=%s!' '%s'  '%s'" %
-                       (temp_file, tmp_config_file, config_file))
+        process.system("sed -i -e 's!^FSDIR=.*$!FSDIR=%s!' "
```

Comment:
@nasastry I would still prefer using python file operations to do it.

Reply:
@narasimhan-v 
To replace three variables `sed` is the easy way. In the above command I am not using any temporary files and it's just a single command.
In python need to open two files, In a loop compare each and every line with the required variables to change, replace them, if there are multiple matches need to handle them, handle the file related exceptions.

Reply:
@nasastry Yes, but still the processing time for shell commands running through python is higher than python native operations. We may need to consider that.